discovery-provider/src/queries/get_health.py

```diff
@@ -116,6 +119,14 @@ def _get_query_insights():
     return query_insights, False
 
 
+def _get_chain_health():
+    try:
+        response = requests.get(CHAIN_HEALTH_ENDPOINT, timeout=0.02)
```

Comment:
this is a really small timeout. is that ok?

Reply:
oh i didn't mean to include this commit haha but yeah it's fine it's consistently below 0.01 even. wanted to make it small to not impact health check